origin-notifications/src/app.js

```diff
@@ -108,11 +108,11 @@ app.get('/', async (req, res) => {
 app.post('/', async(req, res) => {
   const { account, endpoint } = req.body
   // cannot upsert because neither is unique
-  const existing = await PushSubscription.find({
+  const existing = await PushSubscription.findAll({
```

Comment:
@DanielVF why `findAll` here instead of `find`? There shouldn't be more than one, should there? And if there is, it doesn't affect this request. I assumed that `find` would be marginally faster, but just curious.

Reply:
There isn't a `find` in the Sequelize API we are using, so I switched to `findAll`. (http://docs.sequelizejs.com/class/lib/sequelize.js~Sequelize.html). (Turns out there is a `findOne` though.)

Although there should only every be one, in the event of a race condition, multiple identical ones could get inserted and multiple returned. I guess this is a tiny, tiny bit more clear that more than one could be returned. I'm pretty neutral though - you are welcome to change it to `findOne`.

Reply:
I guess no method is a decent reason not to use it. 😕 I must have been reading the wrong docs.